Dockerfile.deploy

```diff
@@ -34,9 +34,7 @@ RUN yum install -y                                          \
         uwsgi-2.0.13.1-2.el7                                \
         uwsgi-plugin-python                                 \
     && yum clean all                                        \
-    && curl -sSL https://bootstrap.pypa.io/get-pip.py | python \
-    && pip install six \
-    && pip install setuptools
+    && curl -sSL https://bootstrap.pypa.io/get-pip.py | python
```

Comment:
That still makes be nervous... 💦

Reply:
Very true… in our non-deploy dockerfile we install pip via `RUN yum install -y python-pip` and later upgrade it explicitly via a requirements file. I think I'm going to change this, our local development approach is TONS more secure. Good catch!